docs/developers/node-tutorial.md

```diff
@@ -0,0 +1,393 @@
+         }
+      ]
+   },
+   "validator_set":{
+      "validators":[
+         {
+            "address":"03F1044A6DF782189C7061FF89146B3D33608F17",
+            "pub_key":"sMcFgSIzlD77eZYgV7H4akyxoHCPc2oIQW05qWEB6b4=",
+            "voting_power":5000,
+            "proposer_priority":-40000
+         },
+         {
+            "address":"04881EB0A0A4C1DB414C708249CEEC2FCF348F3E",
+            "pub_key":"WdqZ8hoyc1HxZCJfQrAGKm2fFJZFg7PngPNGkA1RWXc=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"31711F367349D1BD619BD0A39568A69614B8A048",
+            "pub_key":"pvwSRksq3ekXIiYK7IzjQJ870BxLqEma8zRr9n9VnXI=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"5A253EC2A9AB20AFD48C7BED2AFCA53F5C80BCA6",
+            "pub_key":"RnmnTlKoKxNoh2TpohBDP3cKlx4ATiPOCvQFk/6xpUU=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"79BEB39F4B396F9278DA03F1C97F9BE3B10B12D3",
+            "pub_key":"oh/N+GOIennBOAa/gPNCso1mDlqaHQNn7Op/X8opbeY=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"7F1105B7B219481810C49730AECB1A83036BCA3A",
+            "pub_key":"Ow/AHP/Q3guPGymUKpvhnwae+QoCOpGztpVnP179IG8=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"87265CC17922E01497F40B701EC9F05373B83467",
+            "pub_key":"MNi0Z+uNF5X1Bxj988IDXVl0BKUcLs7LItoMnX6dbg4=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"D345D62BBD18C301B843DF7C65F10E57AB17BD98",
+            "pub_key":"4g3hhdyU4IIgWW/4sR0nax8bsC/M/fDbt1N8s/QanF8=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         },
+         {
+            "address":"DEC2642E786A941511A401090D21621E7F08A36D",
+            "pub_key":"b+Vv6Lcp0bhIjOQncr+OYBHixCvU5+k34y4RqyvpluE=",
+            "voting_power":5000,
+            "proposer_priority":5000
+         }
+      ],
+      "proposer":{
+         "address":"03F1044A6DF782189C7061FF89146B3D33608F17",
+         "pub_key":"sMcFgSIzlD77eZYgV7H4akyxoHCPc2oIQW05qWEB6b4=",
+         "voting_power":5000,
+         "proposer_priority":-40000
+      }
+   },
+   "dah":{
+      "row_roots":[
+         "//////////7//////////uyLCVMJmAItYqbOqgHXm3OwHsq1xQiAX1kZV2Tgcobm",
+         "/////////////////////ykyWNfDJZfigziZC5BN5L00KKuoyDPduwynDywauskL"
+      ],
+      "column_roots":[
+         "//////////7//////////uyLCVMJmAItYqbOqgHXm3OwHsq1xQiAX1kZV2Tgcobm",
+         "/////////////////////ykyWNfDJZfigziZC5BN5L00KKuoyDPduwynDywauskL"
+      ]
+   }
+}
+```
+
+### Submit a PFD Transaction
+
+In this example, e will be submitting a transaction to the PFD endpoint.
+PFD is a PayForData Message.
```

Comment:
we should probably include more info about what a PayForData is (maybe not in this doc, but link it to something that explains well what this tx type is bc it's a core part of celestia and not very easy to understand on the outset).